docs/design/schedule_algorithm.md

```diff
@@ -0,0 +1,39 @@
+
+## Ranking the nodes
+
+The filtered nodes are considered fitable to host the Pod, and it is often that there are more than one nodes remaining. Kubernetes prioritizes the remaining nodes to to find the "best" one for the Pod. The prioritization is performed by a set of priority functions. For each remaining node, a priority function gives a score which scales from 0-10 with 10 representing for "most preferred" and 0 for "least preferred". More over, each priority function is weighted by a positive number and the final score of each node is calculated by adding up all the weighted scores. For example, suppose there are two priority functions, `priorityFunc1` and `priorityFunc2` with weighting factors `weight1` and `weight2` respectively, the final score of some NodeA is:
+
+    finalScoreNodeA = (weight1 * priorityFunc1) + (weight2 * priorityFunc2)
+    
+After the scores of all nodes are calculated, the node with highest score is chosen as the host of the Pod. If there are more than one nodes with equal highest scores, a random one among them is chosen.
+
+Currently, Kuberntes scheduler provides some practical priority functions, including:
+
+- `LeastRequestedPriority`: The node is prioritized based on how much free resource (CPU and Memory) is available on it. The free resource is measured by the capacity minus the sum of the limits of all Pods already on the node, divided by the capacity.
+- `CalculateNodeLabelPriority`: The node can also be prioritized by checking if specified label exists.
+- `BalancedResourceAllocation`: This priority function tries to put the Pod on a node such that the CPU and Memory utilization rate is balanced after the Pod is deployed.
+- `CalculateSpreadPriority`: The Pods behind the same service are spread across the cluster. This is done by giving a higher score to a node if it has fewer Pods behind the same service.
+- `CalculateAntiAffinityPriority`: Similar as the priority function above, this function tries to spread the Pods behind same service among the nodes with particular label.
+
+The details of the above priority functions can be found in [plugin/pkg/scheduler/algorithm/priorities](../../plugin/pkg/scheduler/algorithm/priorities). By default, Kubernetes scheduler uses some of the above priority functions (in [/plugin/pkg/scheduler/algorithmprovider/defaults.go](../../plugin/pkg/scheduler/algorithmprovider/defaults.go)). Similar as predicates, you can combine the above priority functions and assign weight factors (positive number) to them as you want.
```

Comment:
Replace the "By default..." sentence with "Kubernetes uses some, but not all, of these priority functions by default. You can see which ones are used by default in plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go."
(note that your current link is wrong -- you left out the /defaults/ part of the path)